ateam_kenobi/src/kenobi_node.cpp

```diff
@@ -108,6 +109,9 @@ class KenobiNode : public rclcpp::Node
   World world_;
   PlaySelector play_selector_;
   InPlayEval in_play_eval_;
+  DoubleTouchEval double_touch_eval_;
+  ateam_common::GameCommand cached_game_command_;
```

Comment:
It doesn't appear that we use this `cached_game_command_` anywhere that I can find

Reply:
huh, looks like cached and prev are not used. Maybe these were leftover from some temporary work.

Reply:
removed